x-pack/plugins/security/server/authentication/api_keys.ts

```diff
@@ -125,6 +126,35 @@ export class APIKeys {
     this.license = license;
   }
 
+  /**
+   * Determines if API Keys are enabled in Elasticsearch.
+   */
+  async areAPIKeysEnabled(): Promise<boolean> {
+    if (!this.license.isEnabled()) {
+      return false;
+    }
+
+    const id = `kibana-api-key-service-test-${uuid.v4()}`;
```

Comment:
**note**: we don't want to accidentally invalidate a key that actually exists, but we also want the ID to be descriptive in case someone inspecting the ES audit logs wants to trace this back. By combining descriptive text with a UUID, we can guarantee uniqueness while still being descriptive.

Reply:
**updated note**: after I wrote this comment, I realized that the key `id` is generated by ES, and not under the user's control (unlike the key's `name`). Therefore, there is no need to generate a unique id using UUID. I've updated this to always just use `kibana-api-key-service-test` instead.